pex/resolver.py

```diff
@@ -35,15 +36,19 @@ class Unsatisfiable(Exception):
   pass
 
 
-class ResolvedDistribution(namedtuple('ResolvedDistribution',
-                                      ['target', 'requirement', 'distribution'])):
-  """A distribution target, requirement and the resolved distribution that satisfies them both."""
+class InstalledDistribution(namedtuple('InstalledDistribution',
+                                       ['target', 'requirement', 'distribution'])):
+  """A distribution target, requirement and the installed distribution that satisfies them both."""
 
   def __new__(cls, target, requirement, distribution):
     assert isinstance(target, DistributionTarget)
     assert isinstance(requirement, Requirement)
     assert isinstance(distribution, Distribution)
-    return super(ResolvedDistribution, cls).__new__(cls, target, requirement, distribution)
+    return super(InstalledDistribution, cls).__new__(cls, target, requirement, distribution)
+
+
+# A type alias to preserve API compatibility for resolve and resolve_multi.
```

Comment:
"...until version X"?

Reply:
Unfortunately not yet. The TODO here anticipates this though:
https://github.com/pantsbuild/pex/pull/970/files#diff-82dd0097879461fc5b03ef52278ba3f8R708-R709